### ISSUE TITLE
NVDA is reading the day name incorrectly inside the date picker.

### ISSUE THREAD

**Issue by Sandhya-aluri** [OfficeDev/office-ui-fabric-react#8352]
### Environment Information 
 
- __Package version(s)__: (^6.154.0) 
- __Browser and OS versions__: (chrome)

### Please provide a reproduction of the bug in a codepen:
 https://codepen.io/sandhya-aluri/pen/RdJVbx
<img width="537" alt="datepicket" src="https://user-images.githubusercontent.com/48669559/54513095-4e4dba80-497c-11e9-9ca6-3d034f3ccd3c.png">

Fix- Please remove the role=gridcell which is displayed in the browser. 

 
#### Expected behavior:
 NVDA must mention the date day and month properly for the user to understand

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (High) 
 
Products/sites affected: (if applicable)

**Comment by khmakoto** [comment]
I've confirmed that the bug is reproing and that NVDA is reading dates twice, with the second time reading them wrong. I'll check if this is a bug on our side or not.

**Comment by KevinTCoughlin** [comment]
What's the association with:

>Fix- Please remove the role=gridcell which is displayed in the browser.

**Comment by Sandhya-aluri** [comment]
@KevinTCoughlin ,I have checked by removing the role=gridcell in the input by inspecting the element. then it worked. role is defined within fabric date picker.

**Comment by khmakoto** [comment]
@Sandhya-aluri I'm unable to reproduce the bug using Narrator and Edge on your provided CodePen. The dates are read fine, with the date day and month mentioned properly.

If you are able to reproduce this with Narrator or if you think we are actually misusing aria related properties, please let me know! If this keeps happening only on NVDA, however, please report this issue against NVDA. Thanks!

**Comment by Sandhya-aluri** [comment]
@khmakoto This is happening in NVDA , will report to them. Thank you

**Comment by khmakoto** [comment]
@Sandhya-aluri for sure! Feel free to ask if anything else comes up! But if that was it, would you mind closing the issue? Thanks!
